app/Http/Requests/Api/V1_0/CreateProjectCustomFieldRequest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Http\Requests\Api\V1_0;
+
+use App\Http\Requests\AbstractJsonRequest;
+use App\Project;
+use Gate;
+
+class CreateProjectCustomFieldRequest extends AbstractJsonRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        $projectId = $this->route('id');
+        $project = Project::findOrFail($projectId);
+
+        return Gate::allows('update', $project);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Gate' in method 'authorize'.](https://www.codacy.com/app/ym-huang0808/VMS/file/2452055257/issues/source?bid=2972826&fileBranchId=3248926#l21)